### PR TITLE
[doc]index

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
query index doc. 
 code pr #1459 #1598

### CODE REVIEW

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+`CREATE INDEX` 用于为已有标签或边类型创建索引。
+
+### 创建单属性索引
+
+```ngql
+nebula> CREATE TAG INDEX player_index_0 on player(name);
+```
+
+上述语句在所有标签为 _player_ 的顶点上为属性 _name_ 创建了一个索引。
```
Comment (wilsonyou) [inline docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md]: 点

File: docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+```ngql
+nebula> CREATE EDGE INDEX follow_index_0 on follow(degree);
+```
+
+The above statement creates an index for the _degree_ property on all edges carrying the _follow_ edge type.
+
+### Create Composite Index
+
+The schema indexes also supports spawning over multiple properties. An index on multiple properties for all vertices that have a particular tag is called composite index. Consider the following example:
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md]: support

is called a composite index

File: docs/manual-EN/2.query-language/4.statement-syntax/3.utility-statements/describe-syntax.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 DESCRIBE SPACE <space_name>
 DESCRIBE TAG <tag_name>
 DESCRIBE EDGE <edge_name>
+DESCRIBE {TAG | EDGE} INDEX <index_name>
 ```
 
 The DESCRIBE keyword is used to obtain information about space, tag, edge structure.
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/4.statement-syntax/3.utility-statements/describe-syntax.md]: and edge structure

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
+# Schema 索引
+
+```ngql
+CREATE {TAG | EDGE} INDEX [IF NOT EXISTS] <index_name> ON {<tag_name> | <edge_name>} (prop_name_list)
+```
+
+schema 索引可用于快速处理图查询。**Nebula Graph** 支持两种类型的索引：**标签索引**和**边类型索引**。
```
Comment (darionyaphet) [inline docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md]: `边索引`  is better ？

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+多数图查询都从拥有共同属性的同一类型的点或边开始遍历。schema 索引使得这些全局检索操作在大型图上更为高效。
+
+一般地，在使用 `CREATE TAG/EDGE` 语句将标签或边类型创建好之后，即可为其创建索引。
+
+## 创建索引
+
+`CREATE INDEX` 用于为已有标签或边类型创建索引。
```
Comment (darionyaphet) [inline docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md]: Tag 和 Edge 不用翻译吧

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md (hunk 1 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md]: the two are the same thing

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md (hunk 4 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md]: 边索引 is definitely better, but 边类型索引 is more clear to me.

**Pull request merged**